Doc/library/sqlite3.rst

```diff
@@ -868,21 +876,27 @@ Cursor Objects
 
    .. method:: execute(sql[, parameters])
 
-      Executes an SQL statement. Values may be bound to the statement using
+      Execute an SQL statement. Values may be bound to the statement using
       :ref:`placeholders <sqlite3-placeholders>`.
 
       :meth:`execute` will only execute a single SQL statement. If you try to execute
       more than one statement with it, it will raise a :exc:`ProgrammingError`. Use
       :meth:`executescript` if you want to execute multiple SQL statements with one
       call.
 
+      If :attr:`isolation_level` is not :const:`None`,
```

Comment:
```suggestion
      If :attr:`Connection.isolation_level` is not :const:`None`,
```

This is an attribute of `Connection`, not `Cursor`, and so the link doesn't resolve.

Reply:
Ah, I was under the impression that it would magically find its way :) I'll be more explicit.